Tasks/GitHubReleaseV0/task.json

```diff
@@ -35,7 +35,8 @@
             "required": true,
             "helpMarkDown": "Specify the name of the GitHub repository in which GitHub releases will be created.",
             "properties": {
-                "DisableManageLink": "True"
+                "DisableManageLink": "True",
```

Comment:
validate error when user edits and provides invalid repository

Reply:
are you tracking the 'searchable' suggestion as a bug?

Reply:
Added task to track searchable suggestion and showing valid error message issue.